calico-enterprise/threat/web-application-firewall.mdx

```diff
@@ -65,132 +64,178 @@ kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"policySync
 
 ## How to
 
-- [Configure a cluster for WAF](#configure-a-cluster-for-waf)
-- [Add and edit rules](#add-and-edit-rules)
-- [Seeing WAF events in Kibana and Manager UI](#seeing-waf-events-in-kibana-and-manager-ui)
-  - [Kibana](#kibana)
-  - [Manager UI](#manager-ui)
+- [Enable WAF](#enable-waf)
+- [View WAF events](#view-waf-events)
+- [Create global alerts](#create-global-alerts)
+- [Manage WAF](#manage-waf)
+   - [Update rules](#update-rules)
+   - [Disable WAF](#disable-waf)
```

Comment:
Disable WAF for a service and disable feature content is missing.

Reply:
Done

Reply:
Done